tasks/tests/unit/test_save_commit_measurements.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from database.tests.factories.core import CommitFactory, OwnerFactory, RepositoryFactory
+from tasks.save_commit_measurements import SaveCommitMeasurementsTask
+
+
+class TestSaveCommitMeasurements(object):
```

Comment:
[nit] Add a test for the failure too, where the task raises some exception

Reply:
Ah yes, I missed this 🤦, ty